selfdrive/car/subaru/carcontroller.py

```diff
@@ -10,16 +10,23 @@ def __init__(self, dbc_name, CP, VM):
     self.apply_steer_last = 0
     self.frame = 0
 
-    self.es_lkas_state_cnt = -1
-    self.es_distance_cnt = -1
-    self.es_dashstatus_cnt = -1
-    self.infotainmentstatus_cnt = -1
     self.cruise_button_prev = 0
     self.last_cancel_frame = 0
 
     self.p = CarControllerParams(CP)
     self.packer = CANPacker(DBC[CP.carFingerprint]['pt'])
 
+  def should_send_message(self, message_name):
```

Comment:
let's do what other car ports do and just check the freq in the if statement, we can do global clean ups like this later

Reply:
done